src/lib/viewers/box3d/model3d/Model3DViewer.js

```diff
@@ -28,19 +31,27 @@ const LOAD_TIMEOUT = 180000; // 3 minutes
  * This is the entry point for the model3d preview.
  */
 class Model3DViewer extends Box3DViewer {
+    animationClips = [];
+
     /** @property {Object[]} - List of Box3D instances added to the scene */
     instances = [];
 
+    isAnimationPlaying = false;
+
     /** @property {Object} - Tracks up and forward axes for the model alignment in the scene */
     axes = {
         up: null,
         forward: null,
     };
 
+    useReactControls = false;
+
     /** @inheritdoc */
     constructor(option) {
         super(option);
 
+        this.useReactControls = !!this.options.useReactControlsModel3D;
```

Comment:
Do we need a second copy of this value if it's already available on `this.options`?